kuksa-client/README.md

```diff
@@ -27,11 +27,11 @@ After you have installed the kuksa-client package via pip you can run the test c
 $ kuksa-client
 ```
 
-With default CLI arguments, the client will try to connect to a local VISS server e.g. `kuksa-val-server`.
-If you wish to connect to a gRPC server e.g. `kuksa-databroker`, you should instead run:
+With default CLI arguments, the client will try to connect to a local databroker, e.g. a server supporting the `kuksa.val.v1` protocol..
+If you wish to connect to a VISS server e.g. `kuksa-val-server`, you should instead run:
 
 ```console
-$ kuksa-client --ip 127.0.0.1 --port 55555 --protocol grpc
+$ kuksa-client ws://127.0.0.1:8090
```

Comment:
show ws connection and grpc connection command. currently I only see ws connection